.eslintrc.js

```diff
@@ -24,6 +24,18 @@ module.exports = {
         // It's disabled here, but we should using it sparingly.
         "react/jsx-no-bind": "off",
         "react/jsx-key": ["error"],
+
+        "no-restricted-properties": [
+            "error",
+            ...buildRestrictedPropertiesOptions(
+                ["window.innerHeight", "window.innerWidth", "window.visualViewport"],
+                "Use UIStore to access window dimensions instead.",
+            ),
+            ...buildRestrictedPropertiesOptions(
+                ["*.mxcUrlToHttp", "*.getHttpUriForMxc"],
+                "Use Media helper instead to centralise access for customisation.",
+            ),
```

Comment:
these wouldn't catch someone being naughty and doing something like `cli["mxcUrlToHttp"]`

Reply:
Yeah, true... but not sure how much we expect that to happen either...? We can do more exhaustive analysis to also catch those via a custom rule if desired, but this is not really meant to truly _prevent access_, but rather ensure people generally do the right thing.

It seems quite unlikely someone would try to add a call in that style...

Reply:
yea, this feels like it'd be caught in general code review and followed up with a linter complaint